uyuni/common-libs/uyuni-common-libs.spec

```diff
@@ -65,8 +65,6 @@ BuildRequires:  python-devel
 %else
 BuildRequires:  python2-devel
 %endif
-Conflicts:      spacewalk-backend-libs
-Conflicts:      spacewalk-usix
```

Comment:
Can you shortly explain why we can't have obsoletes/provides?

Reply:
uyuni-common-libs is not a 100% replacement for spacewalk-backend-libs. If you just replace it your software using spacewalk-backend-libs will not work. You need to change your code to make it work with uyuni-common-libs.
This is the reason why we cannot define it as Provides/Obsoletes.

Reply:
Right. Problem here is that the packages depending on the old spacewalk-backend-libs and spacewalk-usix won't notice the change, and you fear that maybe some users/customers will install uyuni-common-libs without updating everything else.

Well, in this case I think you are right, only alternative is leaving spacewalk-backend-libs and spacewalk-usix orphaned and add a note about it at the release notes.